src/game.rs

```diff
@@ -54,18 +50,21 @@ impl Game {
 
     pub fn run(&mut self, window: &mut PistonWindow, opengl: &mut GlGraphics, window_size: &Size) {
         while let Some(event) = window.next() {
-            Self::wrap_position(&mut self.position, window_size);
-
+            //Self::wrap_position(&mut self.position, window_size);
```

Comment:
We remove this dead code and the functions that implement it since your new code is much more capable.